p5js/YOLO/YOLO_single_image/sketch.js

```diff
@@ -6,44 +6,45 @@ let objects = [];
 let status;
 
 function setup() {
-
-  createCanvas(400, 266);
-  img = createImg('images/turtle.png', imageReady);
+  createCanvas(640, 420);
+  img = createImg('images/cat2.JPG', imageReady);
```

Comment:
Would `loadImage()` in `preload()` work? If not, we can leave as is but we should file an issue to support `loadImage()` with YOLO!

Reply:
I think I tried to use `loadImage` in `preload` but then was getting the `tf.js error - cannot x, y, z with empty pixels` error so I opted for the above for now. It'll be great though to get preload working here!

Reply:
Is it `preload()` that is causing the problem or `loadImage()`? My suspicion is that the `YOLO` implementation is failing with any `p5.Image` (unrelated to `preload()` This is something that @NHibiki looked at for other models, but perhaps we missed this one? Should I file a new bug report?

Reply:
I could be wrong, but when I try either `preload()` or `loadImage()`, my `p5 image pixels[]` array seems to be empty which would lead me to believe that pixels aren't being loaded? 

I did 2 tests here:

## with loadImage in preload()

The loadImage() in preload takes much longer to fail than the callback version which would indicate to me that preload is doing its thing, but the image passed to YOLO isn't what YOLO expects. Maybe it has to do with the the p5 dom createImg element that YOLO is using?

<img width="1210" alt="Screen Shot 2019-03-18 at 09 29 29" src="https://user-images.githubusercontent.com/3622055/54533492-b2628580-4960-11e9-9da4-9348122dca16.png">

<img width="1316" alt="Screen Shot 2019-03-18 at 09 29 46" src="https://user-images.githubusercontent.com/3622055/54533493-b2628580-4960-11e9-8ec1-149edc0d2a82.png">


## with loadImage with a callback

<img width="1316" alt="Screen Shot 2019-03-18 at 09 30 24" src="https://user-images.githubusercontent.com/3622055/54533495-b2628580-4960-11e9-874b-c8b0d69b9c57.png">
<img width="1210" alt="Screen Shot 2019-03-18 at 09 30 27" src="https://user-images.githubusercontent.com/3622055/54533496-b2fb1c00-4960-11e9-85a8-1ef9df47191b.png">